providers/aws/resources/aws.lr

```diff
@@ -80,6 +80,8 @@ private aws.vpc @defaults("id isDefault cidrBlock region") {
   serviceEndpoints() []aws.vpc.serviceEndpoint
   // List of peering connections associated with the VPC
   peeringConnections() []aws.vpc.peeringConnection
+  // Internet gateway blocking mode or off if VPC Block Public Access (VPC BPA) is not enabled
```

Comment:
So the value is `off` if VPC BPA isn't enabled... what are other values?

Reply:
Should be these strings I think: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcblockpublicaccessoptions.html

Reply:
```suggestion
  // Internet gateway blocking mode: block-bidirectional, block-ingress, or off
```